### DISCUSSION TITLE
You/Your vs We/Our

### DISCUSSION THREAD

**Discussion by Asartea** [TheOdinProject/curriculum#26965] [Ideas]
Currently, TOP (in my experience), mostly uses you/your when addressing a learner. Ocassionally however, (part of an) lesson will suddenly switch to we/our instead, which is jarring and looks weird. Take as an example this extracts from the Battleship lesson (emphasis mine):

> Begin **your** app by creating the Ship class/factory (your choice).
    **Your** ‘ships’ will be objects that include their length, the number of times they’ve been hit and whether or not they’ve been sunk.
    REMEMBER **you** only have to test your object’s public interface. Only methods or properties that are used outside of **your** ‘ship’ object need unit tests.
    Ships should have a hit() function that increases the number of ‘hits’ in **your** ship.
    isSunk() should be a function that calculates whether a ship is considered sunk based on its length and the number of hits it has received.
Create Gameboard class/factory.
    Note that **we** have not yet created any User Interface. **We** should know **our** code is coming together by running the tests. **You** shouldn’t be relying on console.log or DOM methods to make sure **your** code is doing what you expect it to.

As far as I can see TOP currently doesn't specify any rules on which should be used: given the prevalence I propose that you/your becomes the enforced standard

**Comment by JoshDevHub** [comment]
I agree that example you have there should probably switch to you/your.

I think it's unlikely that this rule could be universally applied though. Consider something like this bit from the "Testing Basics" lesson in the JavaScript course:
> This lesson is going to center around __Jest__. The biggest reasons for this decision are that one of the best resources we've found for explaining JavaScript testing uses it and they have fantastic documentation.

Here "we" is appropriate because it's from the voice of the maintainers collectively and the user is actually excluded. Auditing every single occurrence of "we" and considering it in context seems like a lot of work. A quick grep tells me that there are over ~2k instances of it.
